source/content/reducing-large-repos.md

```diff
@@ -33,6 +33,8 @@ You can output the size of your repository by running [`git count-objects -vH`](
 
 ## Prune and Optimize Large Repositories
 
+<Alert title="Note" type="info" >For sites using custom upstream, apply the following steps directly to the custom upstream repository. Force push the updated custom upstream repository branch to the existing sites' Pantheon repository origin.</Alert>
```

Comment:
@marfillaster - is the second sentence here HOW they apply the following steps?  Or is it the first step in their process (to be completed before step 1 below)?